decidim-core/lib/decidim/attachable.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+require "active_support/concern"
+
+module Decidim
+  # A concern with the features needed when you want a model to be able to create
+  # links from it to another resource.
+  module Attachable
```

Comment:
GrammarNaziCI: I find the name `Attachable` a bit misleading. Wouldn't `HasAttachments` or `WithAttachments` wouldn't be more explicit? I would expect an `Attachable` object to be *attached* to something else, not the other way around.